meta/main.yml

```diff
@@ -24,7 +24,7 @@ galaxy_info:
   # this branch. If travis integration is cofigured, only notification for this
   # branch will be accepted. Otherwise, in all cases, the repo's default branch
   # (usually master) will be used.
```

Comment:
If it says " Otherwise, in all cases, the repo's default branch (usually master) will be used", it means master branch doesn't need to be specified, doesn't it? I'm not sure.

Reply:
Yeah, missed that. Thanks.